### ISSUE TITLE
Install command fails

### ISSUE THREAD

**Issue by lowfuel** [Didstopia/satisfactory-server#2]
**Describe the bug**
Steamcmd is hanging. Any ideas?

> docker@satisfactory:/steamcmd/satisfactory$ /steamcmd/linux32/steamcmd +runscript /app/install.txt                          
> Redirecting stderr to '/app/Steam/logs/stderr.txt'                                                                          
> src/tier0/threadtools.cpp (4071) : Probably deadlock or failure waiting for thread to initialize.                           
> [  0%] Checking for available updates...                                                                                    
> Thread failed to initialize                                                                                                 
> src/tier0/threadtools.cpp (4071) : Probably deadlock or failure waiting for thread to initialize.                           
> Thread failed to initialize                                                                                                 
> CWorkThreadPool::StartWorkThread: Thread creation failed.                                                                   
> Exiting on SPEW_ABORT

**Comment by Dids** [comment]
@lowfuel While researching this I did come across others with a Synology NAS and similar issues, right down to the hard locking that happens when it crashes.

If you're up for one more attempt, there's a new image tag that uses Ubuntu 18.04 (instead of 20.04), which just might be enough to fix it, as there's some known issues with `steamcmd` and even barely modern linux distros, but these issues have been around for years and there's still no official fix, or even a statement acknowledging them.

Anyway, the new tag is `didstopia/satisfactory-server:ubuntu-18.04` and as I was able to replicate the thread locking issues on a different machine/server, I also managed to fix it with this new image, so I'm very hopeful about it fixing your issue as well!
